lib/grinder_files.dart

```diff
@@ -98,12 +100,210 @@ class FileSet {
 }
 
 /**
+ * A class to make it easier to manipulate file system entites. Once paths or
+ * entites are converted into `Path`s, they can be easily copied, deleted,
```

Comment:
Hmm. Yeah, what I really want to do is just add additional methods to `FileSystemEntity`. Having `Path` implement that would work. The name `Path` generally also means something else in a dart context (as in, a library prefix for the `path` library). So a different class name like `Entity` might be disambiguating.